core/src/main/java/io/aiven/klaw/service/OperationalRequestsService.java

```diff
@@ -0,0 +1,341 @@
+    return getOperationalRequestModels(operationalRequests);
+  }
+
+  private List<OperationalRequestsResponseModel> getOperationalRequestModels(
+      List<OperationalRequest> operationalRequestList) {
+    List<OperationalRequestsResponseModel> operationalRequestModelList = new ArrayList<>();
+    OperationalRequestsResponseModel operationalRequestModel;
+    String userName = getUserName();
+    Integer userTeamId = commonUtilsService.getTeamId(userName);
+
+    int tenantId = commonUtilsService.getTenantId(userName);
+    List<String> approverRoles =
+        rolesPermissionsControllerService.getApproverRoles("TOPICS", tenantId);
+    List<UserInfo> userList = manageDatabase.getUsersPerTeamAndTenant(userTeamId, tenantId);
+
+    for (OperationalRequest topicReq : operationalRequestList) {
+      operationalRequestModel = new OperationalRequestsResponseModel();
+      copyProperties(topicReq, operationalRequestModel);
+      operationalRequestModel.setRequestStatus(RequestStatus.of(topicReq.getRequestStatus()));
+      operationalRequestModel.setOperationalRequestType(topicReq.getOperationalRequestType());
+
+      operationalRequestModel.setTeamname(
+          manageDatabase.getTeamNameFromTeamId(tenantId, topicReq.getRequestingTeamId()));
+
+      // show approving info only before approvals
+      if (RequestStatus.APPROVED != operationalRequestModel.getRequestStatus()) {
+        operationalRequestModel.setApprovingTeamDetails(
+            updateApproverInfo(
+                userList,
+                manageDatabase.getTeamNameFromTeamId(tenantId, userTeamId),
+                approverRoles,
+                operationalRequestModel.getRequestor()));
+      }
+      operationalRequestModelList.add(setRequestorPermissions(operationalRequestModel, userName));
+    }
+    return operationalRequestModelList;
+  }
+
+  private OperationalRequestsResponseModel setRequestorPermissions(
+      OperationalRequestsResponseModel req, String userName) {
+    log.debug(
+        " My request Status {} and userName {} and requestor {}",
+        req.getRequestStatus(),
+        userName,
+        req.getRequestor());
+    if (RequestStatus.CREATED == req.getRequestStatus()
+        && userName != null
+        && userName.equals(req.getRequestor())) {
+      req.setDeletable(true);
+      req.setEditable(true);
+    }
+
+    return req;
+  }
+
+  private String updateApproverInfo(
+      List<UserInfo> userList, String teamName, List<String> approverRoles, String requestor) {
+    StringBuilder approvingInfo = new StringBuilder("Team : " + teamName + ", Users : ");
+
+    for (UserInfo userInfo : userList) {
+      if (approverRoles.contains(userInfo.getRole())
+          && !Objects.equals(requestor, userInfo.getUsername())) {
+        approvingInfo.append(userInfo.getUsername()).append(",");
+      }
+    }
+
+    return String.valueOf(approvingInfo);
```

Comment:
```suggestion
    return approvingInfo.toString();
```
this would be shorter